query-engine/driver-adapters/js/smoke-test-js/prisma/mysql/schema.prisma

```diff
@@ -1,5 +1,6 @@
 generator client {
-  provider = "prisma-client-js"
+  provider        = "prisma-client-js"
+  previewFeatures = ["jsConnectors"]
```

Comment:
This will be changed later/independent PR?

Reply:
Yes, as mentioned in the PR description: https://github.com/prisma/prisma-engines/pull/4201